content/src/metrics.ts

```diff
@@ -39,6 +39,12 @@ export const metrics = validateMetricsDeclaration({
     labelNames: []
   },
 
+  dcl_content_snapshot_generation_time: {
+    help: 'Histogram of time spent generating full snapshots',
+    type: 'histogram',
+    labelNames: ['failed']
```

Comment:
should the label name be `failed` or something more generic?

Reply:
labels = { failed: "true" | "false" }

If you think a better name please go for it